src/transformers/pipelines/fill_mask.py

```diff
@@ -112,8 +154,15 @@ def __call__(self, *args, targets=None, top_k: Optional[int] = None, **kwargs):
             - **token** (:obj:`int`) -- The predicted token id (to replace the masked one).
             - **token** (:obj:`str`) -- The predicted token (to replace the masked one).
         """
-        inputs = self._parse_and_tokenize(*args, **kwargs)
-        outputs = self._forward(inputs, return_tensors=True)
+        model_inputs = self.get_model_inputs(inputs, *args, **kwargs)
+        self.ensure_exactly_one_mask_token(model_inputs)
+        if isinstance(model_inputs, list):
+            outputs = []
```

Comment:
`outputs` is defined as a tensor here, but `shape` is called on it on line #172, which will not work

Reply:
Fixed most occurrence. Definite culprit was the absence of a no pad_token test. (Reformer is the only one, but it doesn't have a mask_token either, so really we should find a way to get a proper tokenizer for reformer for this test.